sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/PythonUDF.scala

```diff
@@ -34,7 +34,12 @@ object PythonUDF {
     e.isInstanceOf[PythonUDF] && SCALAR_TYPES.contains(e.asInstanceOf[PythonUDF].evalType)
   }
 
-  def isGroupAggPandasUDF(e: Expression): Boolean = {
+  def isGroupedAggPandasUDF(e: Expression): Boolean = {
+    e.isInstanceOf[PythonUDF] &&
+      e.asInstanceOf[PythonUDF].evalType == PythonEvalType.SQL_GROUPED_AGG_PANDAS_UDF
+  }
+
+  def isWindowPandasUDF(e: Expression): Boolean = {
```

Comment:
Hmmmm .. this bit looks a bit odd because the condition is the same but the only the name is different. We should at least need to leave a comment ...

Reply:
@icexelloss, can we maybe do:

```
def isWindowPandasUDF(e: Expression): = isGroupedAggPandasUDF(e)
```

and explain why they can be same ..

Reply:
Sounds good